src/components/NotificationBar/NotificationBar.scss

```diff
@@ -0,0 +1,53 @@
+@import '../../utils/constants.scss';
+
+.notification-bar--notification {
+  &--show {
+    background-color: $white;
+    height: 88px;
+    width: 608px;
+    z-index: 20;
+    position: fixed;
+    bottom: 0;
+    left: 80px;
+    display: flex;
+    transition: all 1s;
+    overflow: hidden;
+    box-shadow: 0px -2px 7px 2px rgba(76, 76, 76, 0.4);
+  }
+
+  &--hide {
+    height: 0;
+    overflow: hidden;
+  }
+}
+
+.notification-bar--info {
+  flex: 65%;
+}
+
+.notification-bar--dismiss {
+  flex: 1;
+  text-align: right;
+  font-size: 30px;
+  padding-top: 24px;
+  padding-right: 24px;
+  color: #47bed8;
+}
+
+.notification-bar--alert {
+  padding-top: 5px;
+  text-align: left;
```

Comment:
is text-align: left not automatic

Reply:
In this case it defaults to center if not specified